api/openapi/auth.yml

```diff
@@ -175,74 +175,23 @@ paths:
           description: Database can't process request.
         "500":
           $ref: "#/components/responses/ServiceError"
-  /domains/{domainID}/enable:
-    post:
-      summary: Enables a domain
-      description: |
-        Enables a specific domain that is identified by the domain ID.
-      tags:
-        - Domains
-      parameters:
-        - $ref: "#/components/parameters/DomainID"
-      security:
-        - bearerAuth: []
-      responses:
-        "200":
-          description: Successfully enabled domain.
-        "400":
-          description: Failed due to malformed domain's ID.
-        "401":
-          description: Missing or invalid access token provided.
-        "403":
-          description: Unauthorized access the domain ID.
-        "404":
-          description: A non-existent entity request.
-        "422":
-          description: Database can't process request.
-        "500":
-          $ref: "#/components/responses/ServiceError"
 
-  /domains/{domainID}/disable:
-    post:
-      summary: Disable a domain
+  /domains/{domainID}/status:
```

Comment:
Why is important to have one endpoint? It looks more self-documenting to me that way, than having sing `status` endpoint - which usually prompt me that it is status indicator (i.e. something I would use with `GET` to fetch the current status), and not a control switch.